packages/authenticator/amplify_authenticator/lib/src/services/amplify_auth_service.dart

```diff
@@ -311,13 +313,24 @@ class AmplifyAuthService implements AuthService {
   }
 
   @override
-  Future<AmplifyConfig> waitForConfiguration() {
-    return Amplify.asyncConfig;
+  Future<AmplifyConfig> waitForConfiguration() async {
+    final timer = Timer.periodic(const Duration(seconds: 10), (timer) {
+      logger.warn(
+        'Amplify is taking longer to configure than expected.'
```

Comment:
I think this reads slightly better

```suggestion
        'Amplify is taking longer than expected to configure.'
```